docs/manual/manual_ifci_control.rst

```diff
@@ -0,0 +1,61 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _controlling_ifci:
+
+**********************************
+IQUART Flight Controller Interface
+**********************************
+
+Vertiq 2306 modules can be controlled with the IQUART Flight Controller Interface (IFCI) when flashed with the pulsing firmware. A specifc IQUART packet has been created to allow for highspeed control of Vertiq modules. A single packet can include all of the control signals for multiple motors connected to the same UART port on a flight controller. Additionally a telemetry packet can be requested from a specific module every time the packet is sent. A description of the packet is shown in the diagram below.
```

Comment:
instead of "when flashed with the pulsing firmware" maybe "while running pulsing firmware" (and if possible maybe even link to the pulsing info)

Reply:
For now we don't have a public link to the pulsing firmware, so I'll correct the grammar.  Once we have a public link I can insert it here. What's the best way to add a 'TODO' in this case?